lib/Auth/Process/PrivacyideaAuthProc.php

```diff
@@ -165,7 +171,7 @@ public function process(&$request): void
         // Check if it should be controlled that user has no tokens and a new token should be enrolled.
         if (!$triggered && !empty($this->authProcConfig['doEnrollToken']) && $this->authProcConfig['doEnrollToken'] === 'true')
         {
-            $stateId = $this->enrollToken($stateId, $username);
+            $stateId = $this->enrollToken($stateId, $username, $headersToForward);
```

Comment:
rename $headersToForward to $headers

Reply:
fixed [f2bb187](https://github.com/privacyidea/simplesamlphp-module-privacyidea/pull/198/commits/f2bb187f584e1a7a8c5c5fec38dee8d19a3e6b78)